### PR TITLE
Also test PHP 7.0 with PostgreSQL

### PR DESCRIPTION
**Pull request by benediktg**
:shipit:

### CODE REVIEW

**Comment by enoch85** [comment]
PostsgreSQL <3

**Comment by MorrisJobke** [comment]
@benediktg Please drop the PHP 7.1 stuff, because it fails. Then we can merge 7.0 and look into why 7.1 fails.

File: .drone.yml [hunk 1]
```diff
@@ -210,6 +210,24 @@ pipeline:
       matrix:
         DB: postgres
         PHP: 5.6
+  postgres-php7.0:
+    image: nextcloudci/php7.0:php7.0-7
```
Comment (MorrisJobke) [inline .drone.yml]: And this should be -13 🙈

**Comment by rullzer** [comment]
Awesome!

But this needs a rebase it seems.

**Comment by benediktg** [comment]
> But this needs a rebase it seems.

Done. ;)

**Review by rullzer — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**